### ISSUE TITLE
Enable picky mode

### ISSUE THREAD

**Issue by Prodeguerriero** [davidlday/vscode-languagetool-linter#756]
**Is your feature request related to a problem? Please describe.**
I always use LT's [Picky Mode](https://languagetool.org/insights/post/picky-mode/) when running the extension in the browser, so now I am used to LT being, well, picky. 😬

**Describe the solution you'd like**
I'd like to be able to enable picky mode for the linter.

**Describe alternatives you've considered**
None, really, besides dealing with it. It's not exactly a dealbreaker.

**Additional context**
N/A

**Comment by davidlday** [comment]
Thanks for opening this feature request!

I think I have it working, but could you please provide a small example of a document that gets different results in picky vs. default mode so I can add it to the testing suite?

**Comment by Prodeguerriero** [comment]
I'm not home right now, but I'll do it asap.

Otherwise, the first rule that comes to mind is to have a comma after
"e.g.".
If you omit the comma, picky mode should flag it, normal mode will not.
--
Gabriele Bartoli
Gender pronouns: he, him, his.

**Comment by Prodeguerriero** [comment]
Nor sure if you still need a sample, but here it is:

```
I need a sentence that will break because of an obvious mistoke.
Then, I require one that includes no mistakes at all. I.e., the perfect sentence.
Finally, one that will trigger picky mode. I.e. this one.
```

**Comment by davidlday** [comment]
Sorry for the delay. Creating an automated test case turned out to be a lot trickier than I expected. I have validated that it works by running the extension and toggling the settings. Will release shortly.

**Comment by Prodeguerriero** [comment]
Thanks a lot, sir.

**Comment by davidlday** [comment]
You're welcome! Let me know if you run into any issues, please!
